impl/src/main/java/com/okta/authn/sdk/impl/resource/DefaultRecoverPassowrdRequest.java

```diff
@@ -1,85 +0,0 @@
-/*
```

Comment:
This file (typo in name) is deleted because it duplicates `DefaultRecoverPasswordRequest.java` which resides in the same package.